spec/requests/movies_spec.rb

```diff
@@ -195,4 +195,59 @@
         expect(movie_error['user_id']).to include("can't be blank")
       end
   end
+
+  describe "PATCH/update" do
+    it 'udates all the movies' do
+      movie = user.movies.create(
+        title: 'born again',
+        genre: 'action',
+        image: 'https://images.unsplash.com/photo-1627465688839-f088954a40a0?auto=format&fit=crop&q=80&w=982&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+        trailer: 'https://www.youtube.com/watch?v=V-yt7-DJZyY&ab_channel=Soundridemusic-NoCopyrightMusic',
+        rating: 4,
+        description: 'Ex Marine who turned his life to the lord has now been called back into action',
+        platform: 'https://www.youtube.com/'
+      )
+
+      get '/movies'
```

Comment:
just for your records, this http request was not required for the purpose of this test